google/cloud/spanner/integration_tests/database_admin_integration_test.cc

```diff
@@ -425,34 +424,38 @@ TEST_F(DatabaseAdminClientTest, CreateWithEncryptionKey) {
                       .get();
   ASSERT_STATUS_OK(database);
   EXPECT_EQ(database->name(), database_.FullName());
-  EXPECT_TRUE(database->has_encryption_config());
-  if (database->has_encryption_config()) {
-    EXPECT_EQ(database->encryption_config().kms_key_name(),
-              encryption_key.FullName());
-  }
+  if (emulator_) {
+    EXPECT_FALSE(database->has_encryption_config());
+  } else {
+    EXPECT_TRUE(database->has_encryption_config());
+    if (database->has_encryption_config()) {
+      EXPECT_EQ(database->encryption_config().kms_key_name(),
+                encryption_key.FullName());
+    }
 
-  auto get_result = client_.GetDatabase(database_);
-  ASSERT_STATUS_OK(get_result);
-  EXPECT_EQ(database->name(), get_result->name());
-  EXPECT_TRUE(get_result->has_encryption_config());
-  if (get_result->has_encryption_config()) {
-    EXPECT_EQ(get_result->encryption_config().kms_key_name(),
-              encryption_key.FullName());
-  }
+    auto get_result = client_.GetDatabase(database_);
+    ASSERT_STATUS_OK(get_result);
+    EXPECT_EQ(database->name(), get_result->name());
+    EXPECT_TRUE(get_result->has_encryption_config());
+    if (get_result->has_encryption_config()) {
+      EXPECT_EQ(get_result->encryption_config().kms_key_name(),
+                encryption_key.FullName());
+    }
 
-  // Verify that encryption config is returned via ListDatabases().
-  auto list_db = [&] {
-    for (auto const& db : client_.ListDatabases(database_.instance())) {
-      if (db && db->name() == database_.FullName()) return db;
+    // Verify that encryption config is returned via ListDatabases().
+    auto list_db = [&] {
+      for (auto const& db : client_.ListDatabases(database_.instance())) {
```

Comment:
For a future PR: sounds like it would be nice to have a helper to say things like:

```
EXPECT_THAT(ToVector(client_.ListDatabases(...)), Contains(IsOkAndHolds(...)));
```

Reply:
Ack.